binr/radare2/radare2.c

```diff
@@ -1319,7 +1319,7 @@ int main(int argc, char **argv, char **envp) {
 			if (has_project) {
 				r_config_set (r.config, "bin.strings", "false");
 			}
-			if (iod && r_core_hash_load (&r, iod->name) == false) {
+			if (iod && r_bin_file_hash_load (r.bin, r.config, iod->name) == false) {
```

Comment:
bad naming, r_bin_file expects to be something using the RBinFile . also _load() expects to parse the headers and such. i would say.. `r_bin_file_hash()` and use the current RBinObject loaded to recompute the checksums.